cristin-import/src/test/java/no/unit/nva/cristin/lambda/CristinPatchEventConsumerTest.java

```diff
@@ -110,14 +102,17 @@ void shouldThrowExceptionWhenSearchingForNvaPublicationByCristinIdentifierReturn
                                                               partOfCristinId);
         var fileUri = s3Driver.insertFile(randomPath(), partOfEventReference);
         var eventReference = createInputEventForFile(fileUri);
-        var input = toInputStream(eventReference);
-        Executable action = () -> handler.handleRequest(input, outputStream, CONTEXT);
-        var exception = assertThrows(ParentPublicationException.class, action);
-        assertThat(exception.getMessage(), containsString(INVALID_PARENT_MESSAGE));
+        var sqsEvent = createSqsEvent(eventReference);
+        handler.handleRequest(sqsEvent, CONTEXT);
+
+        var actualReport = extractActualReportFromS3Client(eventReference,
+                                                           ParentPublicationException.class.getSimpleName(),
+                                                           childPublication.getIdentifier().toString());
+        assertThat(actualReport.getException(), containsString(INVALID_PARENT_MESSAGE));
     }
 
     @Test
-    void shouldThrowExceptionWhenSearchingForNvaPublicationByCristinIdentifierReturnsNoPublication()
+    void shouldStoreErrorReportWhenSearchingForNvaPublicationByCristinIdentifierReturnsNoPublication()
```

Comment:
NoPublication indicates invalid parent?

Reply:
renamed to: shouldStoreErrorReportWhenSearchingForParentPublicationByCristinIdentifierReturnsNoPublication